spec/unit/tariff_synchronizer/base_update_spec.rb

```diff
@@ -40,12 +40,76 @@
     end
   end
 
+  describe '.applicable_download_date_range' do
+    shared_examples_for 'an applicable download date range' do |update_factory|
+      let(:today) { Time.zone.today }
+      let(:pending_issue_date) { today - 21.days }
+      let(:applied_issue_date) { today - 22.days }
+      let(:failed_issue_date) { today - 23.days }
+
+      context 'when choosing a pending update older than the default download from date' do
```

Comment:
what about being more explicit here?
`when a pending update is older than 20 days ago?`

If we update the constant, the tests will break and we need to update them, and the description.

Reply:
I'd rather not update the context description as well as the constant personally - more to maintain.

Reply:
We could stub the constant and then we can change it independent of the test. :thinking: